### DISCUSSION TITLE
Closer collaboration with the CU EarthLab

### DISCUSSION THREAD

**Discussion by kmpaul** [ProjectPythia/pythia-foundations#162] [Ideas]
We are already linking to content in the [CU Earth Lab](https://www.earthdatascience.org/) from the Project Pythia Resource Gallery, but I don't think we've opened a discussion around how better to collaborate with the EarthLab and, in particular, the EarthLab's Earth Data Science learning portal.

It seems to me that there is a lot of "Foundations" content in the EarthLab's learning portal, including introductions to git and GitHub.  I haven't done a very thorough exploration of their educational content, but it looks like a lot of their content is "static" (i.e., HTML pages).  It also looks like much of their content is designed to be run from the "student's" laptop, rather than on the cloud.  And that means their content doesn't really address "Big Data" issues (like ones that require distributed computing).  In general, it seems that the EarthLab content focuses on `matplotlib`, `pandas`, general Python, git, bash, GitHub.  Much of this is in our Foundations Book, so there is overlap.  However, there do appear to be modules/lessons on the EarthLab that *at least demonstrate* the usage of Xarray, Jupyter Notebooks, and even mentions of Dask.  It is unclear to me how much content there is on this.  After the introductory/foundations material, they seem to have taken the approach of focusing on "Use Cases" and examples, akin to our Resource Gallery.

I see the EarthLab's learning resources and Project Pythia's resources evolving in much the same direction as time goes on.  In order to prevent reproduction of the safe efforts across both projects, I believe that we should discuss how EarthLab and Project Pythia can collaborate.  How can we distinguish from each other?  How can we leverage each other's work (other than just providing links to each other's sites)?  There is interest in collaborating on the EarthLab side, so I hope we can match that interest.

**Comment by kmpaul** [comment]
If anyone is interested in seeing what technology the EarthLab has lessons on, or at least potentially has lessons on, you can see the Earth Lab Analytics Python Environment (conda env file) [here](https://github.com/earthlab/earth-analytics-python-env/blob/main/environment.yml).

It is most notable to see what is *missing* from the environment, namely Dask and HvPlot (and all Holoviews/Holovis tools).

**Reply by clyne** [reply]
Is the CU Earth Lab content actively being developed?

**Reply by kmpaul** [reply]
I'm not sure, but I'm talking with Leah Wasser from CU about it tomorrow at noon mountain time.
